File: Whist/src/dev/tgpgamez/whist/engine/Decks.java
package dev.tgpgamez.whist.engine;

import java.util.ArrayList;
import java.util.List;

public class Decks {

    public static List<Card> STANDARD_CARD_GAME = standardCardGame();







    private static List<Card> standardCardGame() {
        List<Card> cards = new ArrayList<>();
        for(Symbol type : Symbol.values()) {
            for (int i = 1; i < 14; i++) {
                cards.add(new Card(type, i));
            }
        }
        return cards;
    }
}

File: Whist-Version-js/Engine/Decks.ts
import { Card } from "./Card";
import { Symbol } from "./Symbol";

export class Decks {
    public static STANDARD_CARD_DECK: Card[] = this.StandardCardDeck();


    private static StandardCardDeck() {
        var cards: Card[] = new Array<Card>();
        for(const symbol in Symbol) {
            for(let i = 1; i < 14; i++) {
                cards.push(new Card(symbol, i));
            }
        }
        return cards;
    }
}